gax-java/gax/src/main/java/com/google/api/gax/rpc/Callables.java

```diff
@@ -115,7 +115,8 @@ public static <RequestT, ResponseT> ServerStreamingCallable<RequestT, ResponseT>
         callable.withDefaultCallContext(
             clientContext
                 .getDefaultCallContext()
-                .withStreamIdleTimeout(callSettings.getIdleTimeout()));
+                .withStreamIdleTimeout(callSettings.getIdleTimeout())
+                .withStreamWaitTimeout(callSettings.getWaitTimeout()));
```

Comment:
Do we have a unit test verifies that we are getting `waitTimeout` from `callSettings` now?

Reply:
Added a test in ServerStreamingCallSettingsTest, similar to idleTimeout.